RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cff.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cfi import particleFlowClusterECAL
+particleFlowClusterECAL.energyCorrector.applyMVACorrections = cms.bool(True)
+particleFlowClusterECAL.energyCorrector.maxPtForMVAEvaluation = cms.double(90.)
+
+from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
+run2_ECAL_2017.toModify(particleFlowClusterECAL,
+                        energyCorrector = dict(srfAwareCorrection = cms.bool(True), maxPtForMVAEvaluation = cms.double(300.)))
+
+particleFlowClusterECALSequence = cms.Sequence(particleFlowClusterECAL)
```

Comment:
it better stay where it was before in particleFlowCluster_cff.py
the main reason is that in that file we need to modify particleFlowClusterECALSequence for the phase-2 era
and the preferred style for application of era modifiers is to make them in the same file where the object is defined.